### ISSUE TITLE
negative fossil emissions

### ISSUE THREAD

**Issue by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [remindmodel/remind#1561]
As of `/p/projects/remind/modeltests/remind/output/SSP2EU-NPi-AMT_2024-01-05_22.08.21/`, REMIND reports negative fossil emissions in the industry sector, e.g.
```
$ awk -F ';' '1 == NR || $0 ~ /CAZ;Emi\|CO2\|Energy\|Demand\|Industry\|Chemicals\|Solids\|\+\|Fossil/' /p/projects/remind/modeltests/remind/output/SSP2EU-NPi-AMT_2024-01-05_22.08.21/REMIND_generic_SSP2EU-NPi-AMT.mif | column -ts ';' | cut -c -145
Model   Scenario        Region  Variable                                                  Unit       2005       2010       2015       2020       
REMIND  SSP2EU-NPi-AMT  CAZ     Emi|CO2|Energy|Demand|Industry|Chemicals|Solids|+|Fossil  Mt CO2/yr  1.2903585  4.2679138  4.3356236  -3.3315255 
```
see also https://github.com/pik-piam/remind2/issues/520

This error tracks through `remind2::reportEmi()` via `EmiIndSubSec < 0` ← `o37_deFeIndSubSec_woNonEn < 0` ← `o37_demFeIndSubSec < vm_demFeNonEnergySector` to https://github.com/remindmodel/remind/blob/3341f41e94961fb22a748006d90a86caa6be979d/modules/37_industry/subsectors/equations.gms#L215-L231 `vm_demFeNonEnergySector` taking `p37_chemicals_feedstock_share` (here 70 %) of `vm_cesIO(feso_chemicals)`, although 68 % of `fesos` (across all industry subsectors) is provided by `sesobio`:

variable/parameter | value
---|---
`vm_cesIO(feso_chemicals)` | 3.05e-3
`vm_demFeNonEnergySector(sesofos,fesos,indst,ETS)` | 2.14e-3
`vm_demFeNonEnergySector(sesobio,fesos,indst,ETS)` | 0
`vm_demFeSector_afterTax(sesofos,fesos,indst,ETS)` | 3.37e-3
`vm_demFeSector_afterTax(sesobio,fesos,indst,ETS)` | 7.26e-3

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
The structure of `vm_demFeNonEnergySector` with respect to `sosofos` and `sesobio` differs in `q_emiTeDetailMkt`
```
vm_demFENonEnergySector(2020,CAZ,sesobio,fesos,indst,ETS)
                (.LO, .L, .UP, .M = 0, 0, +INF, 0)
       -0.1671  q_emiAllMkt(2020,CAZ,co2,ETS)
        1       q37_demFeFeedstockChemIndst(2020,CAZ,fesos,ETS)
       -0.6112  q37_FeedstocksCarbon(2020,CAZ,sesobio,fesos,ETS)
       -1       q37_feedstocksLimit(2020,CAZ,sesobio,fesos,ETS)

…

vm_demFENonEnergySector(2020,CAZ,sesofos,fesos,indst,ETS)
                (.LO, .L, .UP, .M = 0, 0, +INF, 0)
        0.7784  q_emiTeDetailMkt(2020,CAZ,sesofos,fesos,tdfossos,co2,ETS)
       -0.1671  q_emiAllMkt(2020,CAZ,co2,ETS)
        1       q37_demFeFeedstockChemIndst(2020,CAZ,fesos,ETS)
       -0.6112  q37_FeedstocksCarbon(2020,CAZ,sesofos,fesos,ETS)
       -1       q37_feedstocksLimit(2020,CAZ,sesofos,fesos,ETS)
```
where only `vm_demFENonEnergySector(sesofos)`, but not `vm_demFENonEnergySector(sesobio)` is subtracted
https://github.com/remindmodel/remind/blob/3341f41e94961fb22a748006d90a86caa6be979d/core/equations.gms#L537-L540 due to the structure of of `emi2te` and the use of the `enty2` index.
If plastics from fossil carbon do not create emissions, then plastics from biomass (and synfuels in the REMIND structure) create negative emissions, as they sequester carbon, to be stored permanently in landfills or to be burned later on.

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
- [`q_emiTeDetailMkt`](https://github.com/remindmodel/remind/blob/3341f41e94961fb22a748006d90a86caa6be979d/core/equations.gms#L517-L544) defines `vm_emiTeDetailMkt`, subtracting `vm_demFENonEnergySector × pm_emifac` for `sesofos` only
- [`q_emiTeMkt`](https://github.com/remindmodel/remind/blob/3341f41e94961fb22a748006d90a86caa6be979d/core/equations.gms#L571-L611) sums `vm_emiTeDetailMkt` to `vm_emiTeMkt`
- [`q_emiAllMkt`](https://github.com/remindmodel/remind/blob/3341f41e94961fb22a748006d90a86caa6be979d/core/equations.gms#L616-L640) sums `vm_emiTeMkt` to `vm_emiAllMkt`, including `vm_demFENonEnergySector × pm_emifacNonEnergy` for `sesofos` and `sesobio`

-----

- [ ] why do `pm_emifacNonEnergy` and `pm_emifac` differ?
  ```
  $ dumpgdx fulldata.gdx pm_emifac "2020,CAZ,.*fesos"
  2020  CAZ  sesofos  fesos  tdfossos  co2  0.778360712135476
  $ dumpgdx fulldata.gdx pm_emifacNonEnergy "2020,CAZ,.*fesos"
  2020  CAZ  sesobio  fesos  indst  co2  0.167140757746816
  2020  CAZ  sesofos  fesos  indst  co2  0.167140757746816
  ```

**Comment by mellamoSimon** [comment]
Hi Michaja, thank you for the detailed analysis!
>[q_emiTeDetailMkt](https://github.com/remindmodel/remind/blob/3341f41e94961fb22a748006d90a86caa6be979d/core/equations.gms#L517-L544) defines vm_emiTeDetailMkt, subtracting vm_demFENonEnergySector × pm_emifac for sesofos only

that is a mistake indeed. The goal of that term is that the carbon contained in energy carriers that are not combusted (because they are used as feedstocks) is not accounted for as if it were going into the atmosphere. Thus, they should be discounted, regardless of its origin

>why do `pm_emifacNonEnergy` and `pm_emifac` differ? 

because they are used to represent different carbon flows into the atmosphere. While `pm_emifac` is used to represent emissions when fuels are combusted, `pm_emifacNonEnergy` describes process emissions from feedstock processing into chemical products. Coming up with those emission factors meant making some rough assumptions but that is another issue, independent of the accounting.

**Comment by orichters** [comment]
Isn't that fixed with #1576?

**Comment by mellamoSimon** [comment]
jup, thanks
